next/package.json

```diff
@@ -63,7 +63,7 @@
     "@typescript-eslint/eslint-plugin": "^5.62.0",
     "@typescript-eslint/parser": "^5.62.0",
     "autoprefixer": "^10.4.14",
-    "cypress": "^12.17.3",
+    "cypress": "12.17.3",
```

Comment:
Let's fix the cypress version, because we are tied to the available docker image that we are using in lando, and they have an image with node18 and 12.17.3, but the one with 12.17.4 uses node 20 and we are not ready for that.